runtime/src/lib.rs

```diff
@@ -1026,6 +1026,29 @@ impl IsmpRouter for ModuleRouter {
     }
 }
 
+parameter_types! {
+    pub const Plonky2MaxPubsSize: u32 = 262_144;
```

Comment:
Minor: Perhaps we can reduce this, don't envision applications with that many public inputs

Reply:
Clarification: these are viewed here as bytes, not quantitive amount. Do you have in mind appropriate value we can set?

Reply:
Indeed we can set this to a much smaller value, like a size equivalent to 64 public inputs if we want to be really safe